docs/topics/cli-run.adoc

```diff
@@ -63,21 +67,36 @@ The following command analyzes the `com.acme` and `org.apache` packages of the l
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/{mta-cli} --input /path/to/jee-example-app-1.0.0.ear \
+
+$ <{ProductShortName}_HOME>/bin/windup-cli --input /path/to/jee-example-app-1.0.0.ear \
+
     --output /path/to/report-output/ \
     --target eap:7 --target cloud-readiness --packages com.acme org.apache
 ----
 []
 
 [discrete]
+=== Overriding {ProductShortName} properties
+
+To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler` argument on the command line. For example, to use the _Procyon_ decompiler, use the following syntax:
```

Comment:
The default decompiler cannot be overriden in the new version of mta-cli, this line and the example below should be removed

Reply:
This is for MTR only:
```
windup-documentation/docs/cli-guide-mtr/master.adoc:
   32  
   33  // Run the CLI
   34: include::topics/cli-run.adoc[leveloffset=+2]
   35  
   36  // Use OpenRewrite recipes

windup-documentation/docs/cli-guide/master.adoc:
   36  
   37  // Run the CLI
   38: include::topics/mta-cli-run.adoc[leveloffset=+2]
```

MTA changes now in `mta-cli-run.adoc`